test/unit/rules/quotes-test.js

```diff
@@ -146,6 +169,7 @@ generateRuleTests({
     {
       config: 'single',
       template: '<input type="checkbox">',
```

Comment:
Can we add test cases which combinations of different quote types?

```
<img alt="john's house">
```

I believe we would need to escape the quotes inside for this to fix properly:

```
<img alt='john\'s house'>
```

Reply:
Oof, this is an excellent point. We'll actually need escape via html entities: https://html.spec.whatwg.org/dev/syntax.html#attributes-2

Reply:
I think this [would be better fixed in `ember-template-recast`](https://github.com/ember-template-lint/ember-template-recast/issues/698)